src/utils/date_utils.py

```diff
@@ -56,11 +56,17 @@ def create_date_range(start, end, min_accepted=None, max_accepted=None):
     return date_range
 
 
-def get_datetime_from_filename(filename):
+def get_datetime_from_filename(filename, return_type=False):
     try:
-        res = re.search("([0-9]{4}-[0-9]{2}-[0-9]{2})", filename)
+        res = re.search("([iv])([0-9]{4}-[0-9]{2}-[0-9]{2})", filename)
```

Comment:
This will also require that all dates in the filename are preceded by `i` or `v`.